packages/svelte/test/parser/update.js

```diff
@@ -4,6 +4,8 @@
 import * as fs from 'node:fs';
 import glob from 'tiny-glob/sync.js';
 
+const __dirname = new URL('.', import.meta.url).pathname;
```

Comment:
```suggestion
import { fileURLtoPath } from "node:url";
import glob from 'tiny-glob/sync.js';

const __dirname = fileURLToPath(new URL('.', import.meta.url));
```

pathname breaks on windows because of the leading slash